### PR TITLE
Update Deserialization_Cheat_Sheet.md

### PR DESCRIPTION
**Pull request by SPoint42**
Thank you for submitting a Pull Request (PR) to the Cheat Sheet Series. 

> :triangular_flag_on_post: If your PR is related to grammar/typo mistakes, please double-check the file for other mistakes in order to fix all the issues in the current cheat sheet.

Please make sure that for your contribution:

- [ ] In case of a new Cheat Sheet, you have used the [Cheat Sheet template](https://github.com/OWASP/CheatSheetSeries/blob/master/templates/New_CheatSheet.md).
- [ ] All the markdown files do not raise any validation policy violation, see the [policy](https://github.com/OWASP/CheatSheetSeries#editor--validation-policy).
- [ ] All the markdown files follow these [format rules](https://github.com/OWASP/CheatSheetSeries#conversion-rules).
- [ ] All your assets are stored in the **assets** folder.
- [ ] All the images used are in the **PNG** format.
- [ ] Any references to websites have been formatted as [TEXT](URL)
- [ ] You verified/tested the effectiveness of your contribution (e.g., the defensive code proposed is really an effective remediation? Please verify it works!).
- [ ] The CI build of your PR pass, see the build status [here](https://travis-ci.org/OWASP/CheatSheetSeries/pull_requests).

If your PR is related to an issue, please finish your PR text with the following line:

This PR covers issue #<insert number here>.

Thank you again for your contribution :smiley:

### CODE REVIEW

**Comment by SPoint42** [comment]
Cover [PR #271 ](https://github.com/OWASP/CheatSheetSeries/issues/271)

**Comment by mackowski** [comment]
Thanks for it. I will test this solution soon.

File: cheatsheets/Deserialization_Cheat_Sheet.md [hunk 1]
```diff
@@ -142,6 +142,16 @@ public class LookAheadObjectInputStream extends ObjectInputStream {
...
+One other solution is to use the [Apache Commons IO classes](https://commons.apache.org/proper/commons-io/javadocs/api-2.5/org/apache/commons/io/serialization/ValidatingObjectInputStream.html)
+
+```java
+   ValidatingObjectInputStream ois = new ValidatingObjectInputStream(myUntrustedObjectStream);
+   ois.accept(MyTrustedClass.class);
+   MyTrustedClassObject obj = (MyTrustedClassObject) ois.readObject();
+   ois.close();
+```
+
+
```
Comment (mackowski) [inline cheatsheets/Deserialization_Cheat_Sheet.md]:
```suggestion
```

**Comment by mackowski** [comment]
@ThunderSon can you take a look?

**Comment by mackowski** [comment]
@SPoint42 @rbsec I chatted about it with @ThunderSon and we think that maybe we should just add a link to the `Apache Commons IO classes` after [IBM](https://www.ibm.com/developerworks/library/se-lookahead/) link.

First of all other libraries are treated this way so there is no reason for that one to be treated differently and secondly it will  free us from maintaining the code snipped that can be outdated after some time.

What do you think?

**Comment by SPoint42** [comment]
> @SPoint42 @rbsec I chatted about it with @ThunderSon and we think that maybe we should just add a link to the `Apache Commons IO classes` after [IBM](https://www.ibm.com/developerworks/library/se-lookahead/) link.
> 
> First of all other libraries are treated this way so there is no reason for that one to be treated differently and secondly it will free us from maintaining the code snipped that can be outdated after some time.
> 
> What do you think?

@mackowski I agree, this could be the good way to do

**Comment by rbsec** [comment]
@mackowski sounds like a sensible approach.

**Comment by mackowski** [comment]
I see that everyone agrees on that. @SPoint42 do you want to update your PR to reflect this?

**Comment by mackowski** [comment]
@SPoint42 @rbsec @ThunderSon I made the changes and I think that we can merge this one.

**Pull request merged**